ledger/acctupdates.go

```diff
@@ -1452,6 +1448,16 @@ func (au *accountUpdates) postCommit(ctx context.Context, dcc *deferredCommitCon
 		}
 	}
 
+	// clear baking array to let GC collect data
+	const deltasClearThreshold = 1000
+	if offset > deltasClearThreshold {
+		for i := uint64(0); i < offset; i++ {
+			au.deltas[i] = ledgercore.AccountDeltas{}
+			au.roundTotals[i] = ledgercore.AccountTotals{}
```

Comment:
Don't think this will do anything. There is no allocation in `ledgercore.AccountDeltas`.

Reply:
AccountDeltas has 6 pointers on large data structures.

Reply:
Sorry, I meant `ledgercore.AccountTotals`

Reply:
agree, need to remove